docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+- [.NET Core Packages](packages.md) contain the .NET Core runtime and libraries (reference assemblies and implementations). For example, [System.Net.Http](https://www.nuget.org/packages/System.Net.Http/).
+- [.NET Core Metapackages](packages.md) describe various layers and app-models by referencing the appropriate set of versioned library packages.
+
+Distributions
+-------------
+
+You can download .NET Core at the [.NET Core Getting Started](https://www.microsoft.com/net/core) page.
+
+- The *Microsoft .NET Core* distribution includes the CoreCLR runtime, associated libraries, a console application host and the `dotnet` app launcher. It is described by the [`Microsoft.NETCore.App`](https://www.nuget.org/packages/Microsoft.NETCore.App) metapackage.
+- The *Microsoft .NET Core SDK* distribution includes .NET Core and a set of tools for restoring NuGet packages and compiling and building apps. 
+
+Typically, you will first install the .NET Core SDK to get started with .NET Core development. You may choose to install additional .NET Core (perhaps pre-release) builds. The `dotnet new` command will 
+
+Architecture
+============
+
+.NET Core is a cross-platform .NET implementation. The primary architectural concerns unique to .NET Core are related to providing platform-specific implementations for supported platforms.
+
+Environments
+------------
+
+.NET Core is supported by Microsoft on Windows, OS X and Linux. On Linux, Microsoft primarily supports .NET Core running on Red Hat Enterprise Linux (RHEL) and Debian distribution families.
+
+.NET Core currently supports X64 CPUs. On Windows, X86 is also supported. ARM64 and ARM32 are in progress.
+
+The [.NET Core Roadmap](https://github.com/dotnet/core/blob/master/roadmap.md) provides more detailed information on workload and OS and CPU environment support and plans.
+
+Other companies or groups may support .NET Core for other app types and environment.
+
+Designed for Adaptability
+-------------------------
+
+.NET Core has been built as a very similar but unique product relative to other .NET products. It has been designed to enable broad adaptability, to new platforms, for new workloads and with new compiler toolchains. It has several OS and CPU ports in progress and may be ported to many more. Another example is the [LLILC](https://github.com/dotnet/llilc) project, which is an early prototype of native compilation for .NET Core via the [LLVM](http://llvm.org/) compiler.
+
+The product is broken into several pieces, enabling the various parts to be adapted to new platforms on different schedules. The runtime and platform-specific foundational libraries must be ported as a unit. Platform-agnostic libraries should work as-is on all platforms, by construction. To a large degree, this split is what makes the .NET team efficient. The team invests in reducing platform-specific implementations, prefering platform-neutral C# code whenever an algorithm or API can be implemented in-full or in-part that way.
```

Comment:
This sounds like marketing talk and self-promotion. ("...what makes the .NET team efficient..."?). It's also very vague -- what are the several pieces and various parts? Also, is it clear which libraries are platform-specific and which are platform-agnostic? is there a list?